docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: explain_debug
+- Status: in-progress
+- Start Date: 2015-08-10
+- RFC PR: #2047
+- Cockroach Issue:
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
+will cause simple (single table or index) select statements to display the
```

Comment:
As soon as we have multi-table queries, we will want to be able to `EXPLAIN DEBUG` them, so we should probably go ahead and add a `table` field to the output.